src/applications/yellow-ribbon/components/SearchResult/index.jsx

```diff
@@ -88,7 +88,12 @@ export const deriveInstURLLabel = (school = {}) => {
 
   // Show the school's website URL.
   return (
-    <a href={href} rel="noreferrer noopener" target="_blank">
+    <a
+      href={href}
+      onClick={recordEventOnClick(school)}
+      rel="noreferrer noopener"
+      target="_blank"
+    >
```

Comment:
Link that open in a new window need to be announce to screen readers, in this case that would probably be adding `aria-label=""[the link text here] (Opens in a new window)`

Reply:
@timwright12 this is updated, thanks for the heads up!

Reply:
@zacharymorel sorry, not the href in there, but rather the link text

Reply:
@timwright12 Ah, so just asking for clarification, it would looks something like **aria-label="www.usf.edu Opens in a new window"** or **aria-label="usf.edu Opens in a new window"**

Reply:
Whatever text `{toLower(school?.insturl)}` is outputting

Reply:
Should be good to go now!

Reply:
thanks!